docs/tool-configuration/configuration-ansible.md

```diff
@@ -1,9 +1,9 @@
-> ⚠️ Note from the developers: We are currently in the process of moving our documentation and so the below documentation is only partially correct. For more information on this tool please checkout our [plugin documentation](https://github.com/bitops-plugins/ansible).
+> ⚠️ For more information on this tool please checkout [Ansible plugin repository](https://github.com/bitops-plugins/ansible).
 
 # Ansible
 
 ### Example `bitops.config.yaml`, minimum required: 
-```
+```yaml
 ansible:
     cli: {}
```

Comment:
Something seems wrong with this schema..

Reply:
This is my apologies. I just noticed the below YAML isn't a continuation it's a different block entirely.